Solutions/Samsung Knox Asset Intelligence/Analytic Rules/SamsungKnoxMobileDeviceBootCompromise.yaml

```diff
@@ -0,0 +1,31 @@
+id: fae7e371-aee8-4d3f-8311-2255a45a30b3
+name: Knox Mobile Device Boot Compromise
+version: 1.0.0
+kind: NRT
+description: |
+  'When Knox device boot binary is at risk of compromise.'
+severity: High
+status: Available
+requiredDataConnectors:
+  - connectorId: SamsungDCDefinition
+    dataTypes:
+      - Samsung_Knox_System_CL
+tactics:
+  - Persistence
+relevantTechniques:
+  - T1645
+query: |
+  Samsung_Knox_System_CL 
+  | where Name == "BOOT_COMPROMISED_SOFTWARE_BINARY"
+  and MitreTtp has "T1645"
+suppressionEnabled: false
+suppressionDuration: 5h
+incidentConfiguration:
+  createIncident: true
+  groupingConfiguration:
+    enabled: false
+    reopenClosedIncident: false
+    lookbackDuration: 5h
```

Comment:
Please replace 5h with 5H in all the analytic rules

Reply:
This has been corrected now. Thanks!